src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,22 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import jakarta.persistence.EntityNotFoundException;
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class MovieServiceImpl implements MovieService {
+    private final MovieDao movieDao = new MovieDaoImpl(HibernateUtil.getSessionFactory());
```

Comment:
Instantiating a `MovieDao` implementation inside the service is not recommended. Instead, use dependency injection to provide the `MovieDao` instance to the service. This allows for better decoupling and easier testing.